src/app/tool-editor/sections/hints/hint-list.component.ts

```diff
@@ -40,6 +40,9 @@ export class HintListComponent extends ComponentBase {
     @Input()
     public entries: RequirementBaseModel[] = [];
 
+    @Input()
+    public readonly = false;
```

Comment:
`readonly` is passed but not used... this is fine for now considering we're going to rewrite the hint component anyway.